docs/semgrep-app/getting-started-with-semgrep-app.md

```diff
@@ -68,30 +68,40 @@ You are now signed in to Semgrep App.
 
 #### Permissions for GitHub
 
-This section explains why Semgrep App requires specific permissions. Semgrep App requires the following permissions in order to log in through GitHub:
+This section explains Semgrep App permissions. Semgrep App requests the following permissions when you log in through GitHub:
 
 <dl>
     <dt>Verify your GitHub identity</dt>
     <dd>Enables Semgrep App to read your GitHub profile data, such as your username.</dd>
     <dt>Know which resources you can access</dt>
-    <dd>Enables Semgrep App to access and display private repositories you can scan.</dd>
+    <dd>Enables Semgrep App to display private repositories you can scan.</dd>
     <dt>Act on your behalf</dt>
-    <dd>Enables Semgrep App to start or stop scans on the repository and commit files for continuous integration.</dd>
+    <dd>Enables Semgrep App to perform certain tasks <strong>only on resources that you choose to share with Semgrep App</strong>. By default, when you log into Semgrep App for the very first time, no resources are shared. See <a href ="https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/authorizing-github-apps">When does a GitHub App act on your behalf?</a> in GitHub documentation.</dd>

```

Comment:
> By default, when you log into Semgrep App for the very first time, no resources are shared.

No resources are shared with the user login authorization, ever, except for viewing username and emails.

And Semgrep never actually acts on the user's behalf. See this screenshot:

<img width="920" alt="CleanShot 2022-12-01 at 12 46 21@2x" src="https://user-images.githubusercontent.com/1060436/205123851-e9a36936-4e43-479b-b0e0-631aa5548e87.png">

If we acted on the user's behalf, the comment would come from @underyx instead of @semgrep-app.

Reply:
Have worded that we don't use this capability ever. (This would be easier if GitHub didn't call it a permission...)